tests/test_accounts.py

```diff
@@ -88,7 +88,7 @@ def test_generate_account_from_seed():
 
     assert account == {
         'address': (u'xrb_3i1aq1cchnmbn9x5rsbap8b15akf'
-                     'h7wj7pwskuzi7ahz8oq6cobd99d4r3b7'),
+                    'h7wj7pwskuzi7ahz8oq6cobd99d4r3b7'),
```

Comment:
I think it's better to keep these lined up as before, it makes it easier to compare.

Reply:
done